Ym2149Synth/SynthController.cpp

```diff
@@ -178,6 +178,10 @@ void SynthControllerClass::onControlChange(MidiCallbackClass * midi)
                 for(uint8_t i=0;i!=Patch[synth].patchSize;i++) {
                     midi->sendControlChange(midi->getChannel(), i+1, buffer[i]);
                 }
+                /* Send 5 dummy bytes to pad CC data */
```

Comment:
That was added because the teensy would accept an 11 byte preset, but reading it back out required 16 bytes. Not super sure on the details, but it seemed like the editor was expecting 16 bytes.